### ISSUE TITLE
DataViews: Patterns are not bulk selectable, bulk select checkbox has "Select item: [object Object]" as its aria-label

### ISSUE THREAD

**Issue by talldan** [WordPress/gutenberg#63725]
### Description

The inability to bulk select seems to be an issue in the gutenberg plugin.

<s>The aria-label issue is something I can also see in WP 6.6</s>

### Step-by-step reproduction instructions

1. Navigate to the patterns screen
2. Go to My Patterns
3. Observe that the bulk select checkbox is enabled, but clicking it does nothing
4. Inspect the checkbox using the browser dev tools
5. Observe that the aria-label is "Select item: [object Object]"

### Environment info

Mac OS / Brave

### Please confirm that you have searched existing issues in the repo.

- [X] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [X] Yes

**Comment by ntsekouras** [comment]
I inspected in 6.6 without GB and `aria-labels` seemed fine too for me. Am I missing something?

**Comment by talldan** [comment]
> I inspected in 6.6 without GB and aria-labels seemed fine too for me. Am I missing something?

On a second check, it seems you're right. 👍

**Comment by talldan** [comment]
Just checked (because I definitely am not going crazy) and the aria-label was fixed by https://github.com/WordPress/gutenberg/pull/63710. I guess it was a plugin only issue 🤷
